pkg/agent/openflow/network_policy.go

```diff
@@ -1908,6 +1908,18 @@ func parseMetricFlow(flowMap map[string]string) (uint32, types.RuleMetric) {
 	return parseAllowFlow(flowMap)
 }
 
+func verifyMetricFlow(flow string) bool {
+	dropIdentifier := "reg0"
```

Comment:
What about using this format?

```
func verifyMetricFlow(flow string, keys ...string) bool {
        for _, k := range keys {
		if !strings.Contains(flow, key) {
			return false
		}
	}
        return true
}
```

Reply:
Maybe you can also add `metricFlowIdentifier` into the keys? Then we could use a unique validation.

Reply:
If the string doesn't contain "ct_label" but contains "reg0", it should still return true in this function. I think this format can't implement this logic? @wenyingd

Reply:
No, it returns  false if any key is not contained.